spec/plainyearmonth.html

```diff
@@ -629,6 +629,8 @@ <h1>
         1. Set _other_ to ? ToTemporalYearMonth(_other_).
         1. Let _calendar_ be _yearMonth_.[[Calendar]].
         1. If ? CalendarEquals(_calendar_, _other_.[[Calendar]]) is *false*, throw a *RangeError* exception.
+        1. If _yearMonth_.[[ISOYear]] = _other_.[[ISOYear]] and _yearMonth_.[[ISOMonth]] = _other_.[[ISOMonth]] and _yearMonth_.[[ISODay]] = _other_.[[ISODay]], then
+          1. Return ! CreateTemporalDuration(0, 0, 0, 0, 0, 0, 0, 0, 0, 0).
```

Comment:
Contrary to what I wrote in the commit message, this optimization should be done on the other types as well - the fast path through DifferenceDate is not as effective (although still good to do additionally!) because you have to look up calendar methods before calling DifferenceDate.

Reply:
(Note to self: this refers to the commit "Normative: Fast-path PlainYearMonth difference between identical objects")